generate_bq2_queries.qmd

```diff
@@ -101,8 +103,8 @@ select      <- paste(parts_vars,collapse=",\n")
 query       <- glue("SELECT {select} 
                      FROM  `{project}.FlatConnect.participants_JP`  
                      WHERE d_512820379 = '197316935' )")
-# write.table(query,file=paste(outputpath,"Query_parts_verified_",
-#             currentDate, ".txt", sep=""))
+outpath = "/Users/sansalerj/Desktop/bq2/bq2/original_bq2_query_gen/"
```

Comment:
It looks like you hardcoded an new output path that is specific to your local filesystem. This is not recommended, but since this file is going to be deprecated, I'm not sure it matters.

Reply:
eek yes sorry! i was testing the query output locally. forgot to remove this local path